config/cloud.cfg.tmpl

```diff
@@ -146,6 +146,8 @@ cloud_final_modules:
  - phone-home
  - final-message
  - power-state-change
+ - reset_rmc
+ - refresh_rmc_and_interface
```

Comment:
move these up to before rightscale_userdata. thats really the cutoff for modules.